apps/storybook/src/overlays/Dialog.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import {
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  DialogTitle,
+  Flex,
+} from "@optiaxiom/react";
+import { useState } from "react";
+
+const meta: Meta<typeof Dialog> = {
+  component: Dialog,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Dialog>;
+
+const DialogTemplate = ({
+  content,
+  size,
+}: {
+  content: string;
+  size: "lg" | "md" | "sm";
+}) => {
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  return (
+    <Flex>
+      <Button onClick={handleOpen}>Open Dialog</Button>
+      <Dialog onOpenChange={handleClose} open={open} size={size}>
+        <DialogTitle>Dialog</DialogTitle>
+        <DialogBody>{content}</DialogBody>
+        <DialogFooter>
+          <Button appearance="secondary" onClick={handleClose}>
+            Close
+          </Button>
+          <Button appearance="primary" onClick={handleClose}>
+            Confirm
+          </Button>
+        </DialogFooter>
+      </Dialog>
+    </Flex>
+  );
+};
+
+const LongContent =
+  "This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.This is a longer piece of content that demonstrates how the AlertDialog handles more text. It might wrap to multiple lines depending on the width of the dialog.";
+
+export const Default: Story = {
+  render: () => (
+    <DialogTemplate
```

Comment:
let's forward the args here so we can use storybook controls
```suggestion
  render: (args) => (
    <DialogTemplate
      {...args}
```